content/plugins/gcp.md

```diff
@@ -24,6 +24,39 @@ Be aware that some services and resources vary in availability between regions a
 The GCP plugin uses the official [Google API Python Client](https://github.com/google/google-api-python-client).
 
 
+# GCP Plugin Configuration
+The GCP plugin requires credentials and endpoint setup information in order to authenticate and interact with Google Cloud Provider.
+
+### Providing Credentials as Secrets
+
+ It is recommended that you store your credentials as [secrets]({{< relref "manager/using-secrets.md" >}}). You can do this using the [CLI]({{< relref "cli/secrets.md" >}}).
+ Secrets can then be accessed inside your blueprints, as follows:
+
+ {{< gsHighlight  yaml  >}}
+ external_network:
+    type: cloudify.gcp.nodes.Network
+    properties:
+      gcp_config:  
+        username: { get_secret: keystone_username }
+        password: { get_secret: keystone_password }
+        tenant_name: { get_secret: keystone_tenant_name }
+        auth_url: { get_secret: keystone_url }
+        region: { get_secret: region }
+ {{< /gsHighlight >}}   
+
+
+### Providing Credentials as Environment Variables that are not Stored as Secrets
+If you do not use secret storage, you must provide the following credentials as environment variables:
+
+{{< gsHighlight  yaml  >}}
+       gcp_config:  
+        username: { keystone_username }
```

Comment:
(though actually we should probably just remove this bit along with the secrets bit until GCP-41 is done)